### ISSUE TITLE
Administration: Add a confirmation on profile field delete button

### ISSUE THREAD

**Issue by funkycram** [humhub/humhub#6061]
### Is your feature request related to a problem? Please describe.

In `/admin/user-profile/edit-field?id=x`, if you click on the "Delete" button, the profile field is deleted without confirmation.

### Describe the solution you'd like

Add a confirmation modal box

**Comment by funkycram** [comment]
https://github.com/humhub/humhub/pull/6062

**Comment by luke-** [comment]
There seems to be a problem with the tests, according to the PR. Can you please check this?

I think the additional `options` attribute should be ok. However, I hope we can get rid off the old `HForm` soon...

**Comment by funkycram** [comment]
The error is with `ProfileCest`:
1) ProfileCest: Ensure that profile settings are displayed correctly.
 Test  ../../user/tests/codeception/acceptance/ProfileCest.php:testProfileSettings

  [Facebook\WebDriver\Exception\NoSuchElementException] no such element: Unable to locate element: {"method":"css selector","selector":"#status-bar .success"}

I don't think it can be related to the PR.

**Comment by luke-** [comment]
Hmm, I just reran the test on develop and without your change, the Tests are green:
![image](https://user-images.githubusercontent.com/4736168/214616428-e59e7314-230f-4122-9889-2349d9a987d7.png)

**Comment by funkycram** [comment]
OK, I went deeper in the error.
It's about the `/admin/user-profile` page which is the one where I added the delete confirmation.
The test is:
```
 139. $I->waitForElementVisible("#status-bar .success",30) at ../../../tests/codeception/_support/AcceptanceTester.php:169
 138. $I->click("Delete") at ../../user/tests/codeception/acceptance/ProfileCest.php:201
```
It's normal that the success is not received after clicking on the Delete button, as we now have a confirm modal dialog.
So you now need to add, after the line https://github.com/humhub/humhub/blob/develop/protected/humhub/modules/user/tests/codeception/acceptance/ProfileCest.php#L201 a "click" on the "Confirm" button:
![image](https://user-images.githubusercontent.com/23310825/214620639-98c9c77c-e144-4abe-a0bc-4202a40fd818.png)
